docs/get-started/get-started.md

```diff
@@ -55,6 +55,19 @@ with Browserify or - [css-loader](https://webpack.github.io/docs/stylesheets.htm
 import 'mapbox-gl/dist/mapbox-gl.css';
 ```
 
+## Enable Right-to-Left Language Support
+
+```js
+import {setRTLTextPlugin} from 'react-map-gl';
+
+setRTLTextPlugin('https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-rtl-text/v0.2.0/mapbox-gl-rtl-text.js');
```

Comment:
the latest is `v0.2.3`:
* https://www.npmjs.com/package/@mapbox/mapbox-gl-rtl-text
* https://docs.mapbox.com/mapbox-gl-js/example/mapbox-gl-rtl-text/